torch_optimizer/__init__.py

```diff
@@ -33,3 +33,29 @@
     'RangerVA'
 )
 __version__ = '0.0.1a11'
+
+
+def get(identifier):
+    """ Returns an optimizer class from a string. Returns `identifier` if it
+    is already callable.
+
+    Args:
+        identifier (Union[str, Callable, None]): the optimizer identifier.
+
+    Returns:
+        torch.optim.Optimizer or None
+    """
+    if identifier is None:
+        return None
+    elif callable(identifier):
+        return identifier
+    elif isinstance(identifier, str):
+        cls = globals().get(identifier)
```

Comment:
Lets not do any globals or fancy but just create simple mapping name -> Optimizer. It is not hard to add new optimiser to the mapping and code 100% more readable.